resources/0repo-config.py.template

```diff
@@ -40,41 +40,13 @@ def upload_public_dir(files, message):
 	input("Press Return when done (edit 0repo-config.py:upload_public_dir() to automate this)")
 
 	# Example2: use a Git repository to sync to the hosting server
-	#
```

Comment:
I don't think removing the comments helps here. Though this does need updating. On modern Git, you just need to put this in the `.git/config`:
```
[receive]
	denyCurrentBranch = updateInstead
```

And I do like to see what changes I'm pushing.

Reply:
I've removed this change from the PR for now, so we can work on improving the example separately.

My thinking was: Managing and hosting repos via GitHub Actions and GitHub Pages (like `apps.0install.net`) is likely going to be a more common scenario in the future that manually configuring Git post receive hooks. This GitHub Actions/Pages combo for self-hosting is also the route that Helm went with it's [Chart Releaser](https://github.com/helm/chart-releaser) tool.

So I wanted to optimize for this use-case, encouraging users to simply uncomment the "Git example" as soon as they've done a `git init` in `public/` and leave the whole `git push` part of the story out of scope (maybe referencing a nice tutorial how to put all the pieces together for a CI setup).